config-bootstrapper/src/main/resources/configs/config-bootstrapper/attribute-service/hypertrace/hypertrace_backend_trace_attributes.conf

```diff
@@ -171,6 +171,15 @@ commands = [
               scope: BACKEND_TRACE,
               sources: [QS],
               type: ATTRIBUTE
+            },
+            {
+              fqn: Backend.Trace.backend_operation,
```

Comment:
```suggestion
              fqn: Backend.Trace.operation,
```

Reply:
No. Our column name is `backend_operation` so it will be like this only.

Reply:
`fqn` are deprecated AFAIK. We don't use them anywhere